src/coreclr/src/System.Private.CoreLib/src/System/Reflection/Emit/CustomAttributeBuilder.cs

```diff
@@ -96,6 +101,9 @@ private bool ValidateType(Type t)
             return t == typeof(object);
         }
 
+        [MemberNotNull(nameof(m_con))]
+        [MemberNotNull(nameof(m_constructorArgs))]
+        [MemberNotNull(nameof(m_blob))]
         internal void InitCustomAttributeBuilder(ConstructorInfo con, object?[] constructorArgs,
```

Comment:
Can this method be folded into constructor? We would not need the annotations at all.

Reply:
I'll check.

Reply:
Yup.